packages/apps/hr/src/features/expenseSlice.js

```diff
@@ -272,6 +309,13 @@ const expenseSlice = createSlice({
         manageTotal: true,
       },
     );
+    builder.addCase(silenceSearchExpense.pending, state => {
+      state.loadingMyExpense = false;
+    });
+    builder.addCase(silenceSearchExpense.fulfilled, (state, action) => {
+      state.myExpenseList = action.payload.myExpense;
+      state.expenseToValidateList = action.payload.expenseToValidate;
```

Comment:
You asked for 'returnTotalWithData' so action.payload.expenseToValidate contains data & total.
Also updating totalNumberExpenseToValidate could be great too